ProcessMaker/processmaker
# QA Observation bugfix/FOUR-17073: Launchpad > auto scroll is active when `process info` is pressed many times

## Solution
- Auto scroll is not called anymore, no matter how many times user clicks on Process Info

## How to Test
1. Login to PM
2. Create some Cases (almost 30 Cases)
3. Go to Processes menu
4. Search and open the process card
5. Press My Cases or My Tasks option
6. Press Process Info (between 6 to 8 times)
7. Auto scroll should not be triggered.

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-17073

## Code Review Checklist
ci:next
